client/me/purchases/manage-purchase/plan-details/index.jsx

```diff
@@ -0,0 +1,61 @@
+/**
+ * External Dependencies
+ */
+import { connect } from 'react-redux';
+import React, { Component } from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal Dependencies
+ */
+import Card from 'components/card';
+import ClipboardButtonInput from 'components/clipboard-button-input';
+import FormFieldset from 'components/forms/form-fieldset';
+import FormLabel from 'components/forms/form-label';
+import QueryPluginKeys from 'components/data/query-plugin-keys';
+import { getPurchase, isDataLoading } from '../../utils';
+import { isJetpackPlan, isFreeJetpackPlan } from 'lib/products-values';
+import { getPluginsForSite } from 'state/plugins/premium/selectors';
+
+class PurchasePlanDetails extends Component {
```

Comment:
This component is to be used strictly for Jetpack plans, does it make sense to rename it to `PurchaseJetpackPlanDetails` or similar?